app/webpacker/css/admin_v3/components/messages.scss

```diff
@@ -27,9 +27,9 @@
 
 .flash-container {
   position: fixed;
-  left: 0;
-  bottom: 0;
-  width: 100%;
+  left: 50%;
+  transform: translateX(-50%);
+  bottom: 3.5rem;
   z-index: $flash-message-z-index;
```

Comment:
I wonder if we could just move the `z-index` rule to the child `.flash`, so the container wouldn't cover anything?

Reply:
Maybe that could have been a solution, too.

By the way, I noticed that the flash content seems to be there twice. Once in the flash block and then in a notice block which is perfectly overlaying the first one. It doesn't really matter right now but I guess that we want to clean that up at some point.

Reply:
Oh dear, does that mean you have to click "dismiss" twice?

Reply:
Yes! I didn't think of that but just tested and you are right. You have to click twice. :facepalm: 

That's another issue, low priority though.